Misc/NEWS.d/next/Library/2023-10-20-03-50-17.gh-issue-83151.bcsD40.rst

```diff
@@ -0,0 +1 @@
+Make closure work on :mod:`pdb`
```

Comment:
I'd flesh this out a bit. This is a really nice fix!

Reply:
Okay I tried to explain it better, but I'm not sure if I did it well. Do you think we need a whatsnews entry for it? I think the news entry itself does not need too much polishing as .. well, not a lot of people are reading it.